addons/web/static/src/js/fields/field_wrapper.js

```diff
@@ -61,7 +66,7 @@ odoo.define('web.FieldWrapper', function (require) {
             return this.Component.fieldDependencies;
         }
         get resetOnAnyFieldChange() {
-            return this.Component.resetOnAnyFieldChange;
+            return this._data.attrs.decorations || this.Component.resetOnAnyFieldChange;
```

Comment:
weird, field wrapper should not implement any logic

Reply:
ha yes you told me about this, i forgot :+1:

Reply:
I already changed it, it's now a getter in AbstractField
not a static field anymore, I think it's better for the moment